src/components/List/List.tsx

```diff
@@ -64,11 +69,15 @@ class List extends UIComponent<any, any> {
   // List props that are passed to each individual Item props
   static itemProps = ['debug', 'selection', 'truncateContent', 'truncateHeader', 'variables']
 
-  renderComponent({ ElementType, classes, rest }) {
+  getDefaultAccessibility() {
+    return this.props.selection ? SelectableListBehavior : ListBehavior
```

Comment:
actually, by just looking on this line there is a strong feeling that this selection between behavior strategy should be part of the accessibility itself - due to that we have `props` as an input and some logic that will provide accessibility attributes as an output. Don't see the reasons to introduce indirection and put behavior selection at this level.